spec/requests/ncr_work_order_spec.rb

```diff
@@ -0,0 +1,20 @@
+describe 'proposals' do
+  include ReturnToHelper
+
+  describe "DISABLE_CLIENT_SLUGS" do
+    with_env_var("DISABLE_CLIENT_SLUGS", "ncr") do
+      it "disallows any request for disabled client_slug" do
+        user = create(:user, client_slug: "ncr")
```

Comment:
oh haha I just saw the one below. AND YOU USED ALMOST EXACTLY THE SAME SLUG. guess we are mind melding

 I wonder if we don't need this specific spec, then? do you think there is a benefit to having both?